platform_plugin_aspects/tests/commands/test_load_test_monitor.py

```diff
@@ -0,0 +1,158 @@
+    """
+    options = [
+        # Test our defaults
+        CommandOptions(
+            options={},
+            expected_logs=[
+                "Clickhouse lag seconds: 1",
+                "Celery queues: LMS 5, CMS 5",
+                "Starting monitor for celery with sleep of 10 seconds",
+            ],
+        ),
+        # Test when sleep time is shorter than the time it takes to gather stats
+        CommandOptions(
+            options={"sleep_time": 0},
+            expected_logs=[
+                "this is greater than the sleep time!",
+            ],
+        ),
+        # Test overriding sleep time
+        CommandOptions(
+            options={"sleep_time": 5},
+            expected_logs=[
+                "Clickhouse lag seconds: 1",
+                "Celery queues: LMS 5, CMS 5",
+                "Starting monitor for celery with sleep of 5 seconds",
+            ],
+        ),
+        # Test explicit celery backend
+        CommandOptions(
+            options={"backend": "celery"},
+            expected_logs=[
+                "Clickhouse lag seconds: 1",
+                "Celery queues: LMS 5, CMS 5",
+                "Starting monitor for celery with sleep of 10 seconds",
+            ],
+        ),
+        # Test redis bus backend
+        CommandOptions(
+            options={"backend": "redis_bus"},
+            expected_logs=[
+                "Clickhouse lag seconds: 1",
+                "Starting monitor for redis_bus with sleep of 10 seconds",
+                "Redis bus queue length: [{'group1': 2}, {'group2': 3}]",
+            ],
+        ),
+        # Test kafka bus backend
+        CommandOptions(
+            options={"backend": "kafka_bus"},
+            expected_logs=[
+                "Clickhouse lag seconds: 1",
+                "Starting monitor for kafka_bus with sleep of 10 seconds",
+                "test [test] Lag: 95",
+            ],
+        ),
+        # Test vector backend
+        CommandOptions(
+            options={"backend": "vector"},
+            expected_logs=[
+                "Clickhouse lag seconds: 1",
+                "Vector received: 10 sent: 10 lag: 0",
+                "Starting monitor for vector with sleep of 10 seconds",
+            ],
+        ),
+    ]
+
+    for option in options:
+        yield option
+
+
+@pytest.mark.parametrize("test_command_option", load_test_command_basic_options())
+def test_monitor_options(test_command_option, caplog):
+    option_combination, expected_outputs = test_command_option
+    patch_prefix = (
+        "platform_plugin_aspects.management.commands.monitor_load_test_tracking"
+    )
+
+    with patch.multiple(
+        f"{patch_prefix}",
+        requests=DEFAULT,
+        settings=DEFAULT,
+        redis=DEFAULT,
+        json=DEFAULT,
+        confluent_kafka=DEFAULT,
+        sleep=Mock(side_effect=KeyboardInterrupt),
+    ) as patches:
+        patches["requests"].post.return_value.json.side_effect = (
```

Comment:
These patches are so ugly, but since they're all used here it seemed to make sense to keep them where they're being used instead of making people hunt around for them.